drone/exec/exec.go

```diff
@@ -227,7 +227,16 @@ func exec(cliContext *cli.Context) error {
 	// disabled in favor of mounting the source code
 	// from the current working directory.
 	if !commy.Clone {
-		comp.Mount, _ = os.Getwd()
+		pwd, _ := os.Getwd()
+		comp.Mount = pwd
+		//Add the new labels that helps looking up the step containers
+		//by names
+		if comp.Labels == nil {
+			comp.Labels = make(map[string]string)
+		}
+		comp.Labels["io.drone.pipeline.dir"] = pwd
```

Comment:
are these labels meant for drone desktop?  If yes, it might make sense to use `io.drone.desktop` so that we can associate the labels with the correct client and reduce ambiguity.  For example, one might assume that `dir` represents the pipeline working directory, which would not be the case.

Reply:
yeah mostly for now its targetted for the tools and it make sense to rename it